performance/scaling_data_ingestion.rst

```diff
@@ -259,13 +261,13 @@ To load the data, we can download the data, decompress it, filter out unsupporte
 
     CREATE TEMPORARY TABLE prepare_1 (data jsonb);
     
-    /* Load a file directly from Github archive and filter out rows with unescaped 0-bytes */
+    -- Load a file directly from Github archive and filter out rows with unescaped 0-bytes
     COPY prepare_1 FROM PROGRAM
     'curl -s http://data.githubarchive.org/2016-01-01-15.json.gz | zcat | grep -v "\\u0000"'
     CSV QUOTE e'\x01' DELIMITER e'\x02';
     
-    /* Prepare a staging table */
-    CREATE UNLOGGED TABLE stage_1 AS
```

Comment:
Mainly that it might not really be a good idea in Citus. What if the node restarts just before master_append_table_to_shard starts? That would not only give an unexpected result, but might actually trigger a master_append_table_to_shard bugs (empty tables resulting in empty string as min/max values).